math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -906,6 +1110,17 @@ RVec<T> Intersect(const RVec<T>& v1, const RVec<T>& v2, bool v2_is_sorted = fals
 
 /// Return the elements of v1 if the condition c is true and v2 if the
 /// condition c is false.
+///
+/// Example code, at the ROOT prompt:
+/// ~~~{.cpp}
+/// using namespace ROOT::VecOps;
+/// RVec<double> v1 {1., 2., 3.};
+/// RVec<double> v2 {-1., -2., -3.};
+/// RVec<int> c {0, 1, 1};
+/// auto if_c_v1_else_v2 = Where(c, v1, v2);
```

Comment:
```suggestion
/// auto if_c_v1_else_v2 = Where(v1>1, v1, v2);
```

Reply:
`RVec<int>` will nobody write explicitly!

Reply:
In case you agree, don't forget to remove the condition `c`.

Reply:
Here perhaps one need to be more explicit. I would show what c is in any case.